lib/services/devices/deviceService.js

```diff
@@ -522,6 +621,64 @@ function updateEntityNgsi2(deviceData, updatedDevice, callback) {
     }
 }
 
+
+
+/**
+ * Updates the entity representing the device in the Context Broker using NGSIv2.
```

Comment:
```suggestion
 * Updates the entity representing the device in the Context Broker using NGSIv2.
```

Three times :) Please, do an overall check for this.

Reply:
Fixed b41a18a